Sources/SafeDICore/Generators/DependencyTreeGenerator.swift

```diff
@@ -54,7 +54,7 @@ public final class DependencyTreeGenerator {
             for try await generatedRoot in taskGroup {
                 generatedRoots.append(generatedRoot)
             }
-            return generatedRoots.sorted().joined(separator: "\n\n").trimmingCharacters(in: .whitespacesAndNewlines)
```

Comment:
`trimmingCharacters` exists on `NSString`, which doesn't exist in Swift on Linux. Turns out we didn't need this call anyways though.